### PR TITLE
feat: Addition of Event End Date on Public Event Page

### LINKED ISSUE #2966: Public Event Page should show beginning and end date if event is over several days
If an event is taking place over several days the event only shows the beginning date. Please show beginning and end date if the event is happening over several days.

Screenshot from 2019-05-20 10-28-30

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #2966 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 @ritikamotwani @pradeepgangwar Review Please .
PS-:  Please merge #3021 to see the proper function of this

File: app/controllers/public.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ export default Controller.extend({
       return this.get('session.currentRouteName') !== 'public.index';
     }
   }),
+  displayEndDate: computed('model.startsAtDate', 'model.endsAtDate', function() {
+    return (this.get('model.startsAtDate') !== this.get('model.endsAtDate'));
```
Comment (niranjan94) [inline app/controllers/public.js]: Use [`isSame`](https://momentjs.com/docs/#/query/is-same/) to do the comparison 

```suggestion
    return !this.model.startsAtDate.isSame(this.model.endsAtDate, 'minute')
```

Comment (kushthedude) [reply app/controllers/public.js]: @niranjan94 why have you added ‘minute’ in the end of comparison?

Comment (niranjan94) [reply app/controllers/public.js]: @kushthedude so that the comparison is done up until the minute.

in the public page we only show dates up until the minute. 
So, `2019-12-22 12:00:10` and `2019-12-22 12:00:50` should be considered as equal.

**Comment by kushthedude** [comment]
@niranjan94 Have a look now

**Comment by shreyanshdwivedi** [comment]
@kushthedude travis build is failing. Please fix this

**Comment by kushthedude** [comment]
> @kushthedude travis build is failing. Please fix this

@shreyanshdwivedi Sure,Some minor lintink issue , just doing some final tests for Boundary Conditions will finalise in few minutes

**Comment by kushthedude** [comment]
@CosmicCoder96 Please Review

**Comment by shreyanshdwivedi** [comment]
@kushthedude it would be great if screenshots for both desktop amd mobile view are attached here

**Comment by kushthedude** [comment]
@shreyanshdwivedi - :
In Desktop View:

Screenshot 2019-05-29 at 4 45 46 AM

In Mobile View:
Screenshot 2019-05-29 at 4 45 41 AM

**Comment by kushthedude** [comment]
@shreyanshdwivedi @mrsaicharan1 @uds5501 @CosmicCoder96 Please Review

File: app/templates/public.hbs [hunk 2]
```diff
@@ -12,6 +12,9 @@
           <div class="one wide column"></div>
           <div class="fifteen wide column">
             <h4 class="event time">{{header-date model.startsAt model.timezone}}</h4>
+            {{#if displayEndDate}}
+              <h5 class="event time ends">{{t 'To'}} {{header-date model.endsAt model.timezone}}</h5>
```
Comment (shreyanshdwivedi) [inline app/templates/public.hbs]: @kushthedude why aren't you using same class and font size as that of eventStartDate? Any specific reason to male the eventEndDate smaller?

Comment (kushthedude) [reply app/templates/public.hbs]: @shreyanshdwivedi mainly for two reasons
* Fir devices with small screen , same size font is looking ugly and causing an underflow.
* Since in the ‘end date’ there is an extra ‘to’ and if i make it of same size as of start date , second row will have more width than first row which dont looks good

**Comment by kushthedude** [comment]
@CosmicCoder96 Review please

**Comment by uds5501** [comment]
@kushthedude did you try using muted text ? I think that may serve the purpose without any additional CSS.

**Comment by kushthedude** [comment]
@uds5501 That class will change font colour and break evenness in style

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Review Please

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Please Review

File: app/controllers/public.js [hunk 3]
```diff
@@ -7,6 +8,9 @@ export default Controller.extend({
       return this.get('session.currentRouteName') !== 'public.index';
     }
   }),
+  displayEndDate: computed('model.startsAtDate', 'model.endsAtDate', function() {
+    return !(moment(this.model.startsAtDate).isSame(this.model.endsAtDate, 'minute'));
```
Comment (niranjan94) [inline app/controllers/public.js]: Why the additional bracket ?

Comment (kushthedude) [reply app/controllers/public.js]: Just to ensure the `!` negate property ,

**Comment by kushthedude** [comment]
@niranjan94 @CosmicCoder96 Review Please
